CMakeLists.txt

```diff
@@ -66,6 +66,8 @@ if(image_FOUND)
     target_link_libraries(${PROJECT_NAME} PUBLIC image CGAL::CGAL_ImageIO)
 endif()
 
+set_property(GLOBAL PROPERTY JOB_POOLS one_jobs=1 two_jobs=2 three_jobs=3 four_jobs=4)
+set_property(TARGET ${PROJECT_NAME} PROPERTY JOB_POOL_COMPILE four_jobs)
```

Comment:
why those one_job, two_job etc? why not directly
```suggestion
set_property(TARGET ${PROJECT_NAME} PROPERTY JOB_POOL_COMPILE 4)
```
🤔😅

Reply:
YEah sorry, I took the doc example...